spec/gitx/cli/release_command_spec.rb

```diff
@@ -200,5 +202,38 @@
         should meet_expectations
       end
     end
+    context 'when user confirms release and pull request exists with success status with release_label config' do
+      let(:gitx_config) do
+        {
+          'release_label' => 'release-me'
+        }
+      end
+      before do
+        expect(repo).to receive(:workdir).and_return(temp_dir)
+        File.open(File.join(temp_dir, '.gitx.yml'), 'w') do |f|
+          f.puts gitx_config.to_yaml
+        end
+
+        expect(cli).to receive(:yes?).and_return(true)
+        expect(cli).to receive(:label_pull_request).with(having_attributes(number: 10), 'release-me').and_call_original
+        allow(cli).to receive(:authorization_token).and_return(authorization_token)
+
+        expect(executor).to_not receive(:execute).with('git', 'checkout', 'master')
+        expect(executor).to_not receive(:execute).with('git', 'pull', 'origin', 'master')
+        expect(executor).to_not receive(:execute).with('git', 'merge', '--no-ff', '--message', "[gitx] Release feature-branch to master\n\nConnected to #10", 'feature-branch')
+        expect(executor).to_not receive(:execute).with('git', 'push', 'origin', 'HEAD')
+
+        expect(executor).to receive(:execute).with('git', 'checkout', 'feature-branch').ordered
+        expect(executor).to receive(:execute).with('git', 'update').ordered
+        expect(executor).to receive(:execute).with('git integrate').ordered
+
+        VCR.use_cassette('pull_request_does_exist_with_success_status') do
```

Comment:
It seems like this should use a new cassette for this new usecase/config?  Something like:
```
pull_request_does_exist_with_success_status_and_then_add_label
```

Reply:
Sounds great, done!